c7n/resources/dynamodb.py

```diff
@@ -120,6 +121,93 @@ class KmsFilter(KmsRelatedFilter):
     RelatedIdsExpression = 'SSEDescription.KMSMasterKeyArn'
 
 
+@Table.filter_registry.register('check-continuous-backup')
```

Comment:
we can probably drop the prefix `check`, its a filter which implies that's what its checking.

Reply:
Agreed, it is implied! Thanks for bringing up.

Reply:
Updated in d27a188